frontend/src/Student/components/ShowCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link } from 'react-router-dom'
+import PropTypes from 'prop-types'
+import Moment from 'react-moment'
+import FaPlusCircle from 'babel-loader!react-icons/fa/plus-circle'
+import { Button, ButtonGroup, Row, Col } from 'reactstrap'
+
+const Card = styled.div`
+  background-color: #f8f9fa;
+  border-radius: 5px;
+  margin-top: 15px;
+  padding: 10px;
+  width: 100%;
+`
+
+const ShowCard = (props) => (
+  <Card>
+    <Row>
+      <Col>
+        <h2>{props.name}</h2>
+      </Col>
+      <Col className="text-right">
+      <div>
+        <h5>{props.entries ? props.entries.length : 0}/{props.entryCap} submissions</h5>
```

Comment:
With the `defaultProps` below, `props.show.entries.length` will be zero when there are no submissions, so you could simplify this to just `{props.show.entries.length}/{props.show.entryCap}`

Also, see comment below about "Submissions" vs. "Entries"

Reply:
Oh true, good catch

Reply:
So doing so without the ternery causes a null if the entries are completely empty shown here in this error: 
![image](https://user-images.githubusercontent.com/8568993/36386539-0d82cdc6-1564-11e8-9325-13d0a626689e.png)